### PR TITLE
Hot fixes for older upgrade responder version, v0.0.x

### PR DESCRIPTION
**Pull request by PhanLe1010**
* Update vendor and go modules
* Returns all tags set by user in upgrade response and fix duplicated pointer value bug
* Allow multiple versions with the same tag

### CODE REVIEW

File: upgraderesponder/service.go [hunk 1]
```diff
@@ -263,17 +263,21 @@ func (s *Server) GenerateCheckUpgradeResponse(request *CheckUpgradeRequest) (*Ch
 
 	// Only supports `latest` label for now
 	//latestVer, version, err := s.getParsedVersionWithTag(VersionTagLatest)
```
Comment (yasker) [inline upgraderesponder/service.go:250]: We can remove those (with the comment) if it's no longer the case.

File: upgraderesponder/service.go [hunk 2]
```diff
@@ -263,17 +263,21 @@ func (s *Server) GenerateCheckUpgradeResponse(request *CheckUpgradeRequest) (*Ch
...
+	//	logrus.Errorf("BUG: unable to get an valid tag for %v: %v", VersionTagLatest, err)
+	//	return nil, err
+	//}
 	/* disable version dependency reseponse
 	if reqVer.LessThan(latestVer) {
 		resp.Versions = append(resp.Versions, *version)
 	}
 	*/
-	resp.Versions = append(resp.Versions, *version)
+	//resp.Versions = append(resp.Versions, *version)
```
Comment (yasker) [inline upgraderesponder/service.go:261]: ?

File: upgraderesponder/service.go [hunk 3]
```diff
@@ -236,18 +233,6 @@ func respondWithJSON(rw http.ResponseWriter, obj interface{}) error {
 	return err
 }
 
-func (s *Server) getParsedVersionWithTag(tag string) (*semver.Version, *Version, error) {
```
Comment (yasker) [inline upgraderesponder/service.go:239]: This should be in the previous commit.

**Review by yasker — CHANGES_REQUESTED** [review]

File: upgraderesponder/service.go (hunk 1 above)
Comment (PhanLe1010) [reply upgraderesponder/service.go]: Thank you. Fixed

File: upgraderesponder/service.go (hunk 2 above)
Comment (PhanLe1010) [reply upgraderesponder/service.go]: My bad

File: upgraderesponder/service.go (hunk 3 above)
Comment (PhanLe1010) [reply upgraderesponder/service.go:239]: Thank you. Fixed

**Review by yasker — APPROVED** [review]

**Pull request merged**